attributes/network/cddl/14_proof.cddl

```diff
@@ -0,0 +1,25 @@
+proof@param = ()
```

Comment:
How do you request the object you want the proof from?

Reply:
The response is the object you want the proof from. Whatever the response.

Reply:
Right. It makes more sense than having a network endpoint.